js/app.js

```diff
@@ -4,154 +4,191 @@ let username = prompt('Greetings, user. Do you have a preferred name?');
-console.log("Question Two Information Collected.");
-
-
-// WHAT IS HIS FAVORITE PROGRAMMING LANGUAGE? 
-let language = prompt('Is Ken\'s favorite programming language COBOL?');
-let answer3 = language.toUpperCase();
-if (answer3 === "YES" || answer3 === "Y") {
-    //console.log("Oh, that's not right...")
-    alert("Oh, that's not right...");
-} else if (answer3 === "NO" || answer3 === "N") {
-    //console.log("Yes! That's right!");
-    alert("Yes! That's right!");
-} else {   
-    //console.log("Please answer with either a \'Yes\' or a \'No\'.");
-    alert("Please answer with either a \'Yes\' or a \'No\'.");
-}
-//alert('Hmm... ' + language + ' ... (continues scribbling)');
-console.log("Question Three Information Collected.");
-
-
-
-// WHAT IS HIS FAVORITE PROGRAMMING BEVEREDGE?
-let drink = prompt('Is Ken\'s favorite programming drink Everclear?');
-let answer4 = drink.toUpperCase()
-if (answer4 === "YES" || answer4 === "Y") {
-    //console.log("Oh, that's not right...")
-    alert("Oh, that's not right...");
-} else if (answer4 === "NO" || answer4 === "N") {
-    //console.log("Yes! That's right!");
-    alert("Yes! That's right!");
-} else {   
-    //console.log("Please answer with either a \'Yes\' or a \'No\'.");
-    alert("Please answer with either a \'Yes\' or a \'No\'.");
-}
-alert('We agree, then? '+ drink +" works for me...");
-console.log("Question Four Information Collected.");
-
-
-
-// WHAT IS HIS FAVORITE PROGRAMMING SNACK?
-let snack = prompt('Is Ken\'s favorite programming snack Cheez-its?');
-let answer5 = snack.toUpperCase();
-if (answer5 === "YES" || answer5 === "Y") {
-    //console.log("Yes! That's right!");
-    alert("Yes! That's right!");
-} else if (answer5 === "NO" || answer5 === "N") {
-    //console.log("Oh, that's not right...")
-    alert("Oh, that's not right...");
-} else {   
-    //console.log("Please answer with either a \'Yes\' or a \'No\'.");
-   alert("Please answer with either a \'Yes\' or a \'No\'.");
-}
-alert('So, thoughts on snack material... '+ snack + ', huh. Did not expect that. (continues scribbling)');
-console.log("Question Five Information Collected.");
-
-
-// NUMBER GUESSING GAME
-alert('Let\'s play a number guessing game. Pick a number and I\'ll let you know if you are close, you will have 4 guesses.');
-let correctNum = 7
-let numGuessRemain = 4
-
-while (numGuessRemain > 0) {
-    let numGuess = prompt('Pick a number:');
-    if (numGuess == correctNum) {
-        alert("You guessed right!");
-        numGuessRemain = 0
-        break
-    } else if (numGuess > correctNum) {
-        alert("That's too high...");
-    } else if (numGuess < correctNum) {
-        alert("That's too low...");
-    } else {   
-        //console.log("Please answer with either a \'Yes\' or a \'No\'.");
-        alert("Please answer with a number.");
+function popQuiz() {
+    let score = 0;
+
+    // QUESTION ONE: WHERE DOES HE LIVE?
+    function home (score) {
```

Comment:
You're defining this function inside of another function. Try to have function definitions on the top level. Afterwards, you can set your popQuiz definition to run home(score). This will allow you to use your home function outside of the popQuiz function, while right now home(score) doesn't exist outside of popQuiz().